nominees/play-a-song-save-a-dog.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Play A Song - Save A Dog",
+  "aliases": [
+    "TheSound"
+  ],
+  "description": "Free Music Supports Animal Rescue Efforts",
+  "website": "https://www.thesound.com",
+  "license": [
+    {
+      "spdx": "CC-BY-1.0",
+      "licenseURL": "https://creativecommons.org/licenses/by-nc/1.0/"
```

Comment:
Please request submitter to provide a licenseURL that explicitly indicates the CC-BY-1.0 license as part of the digital solutions resources as opposed to pointing to a [CC link](https://creativecommons.org/licenses/by-nc/1.0/)